.travis.yml

```diff
@@ -7,16 +7,16 @@ env:
   matrix:
     #- DISTRIB="ubuntu" PYTHON_VERSION="2.7" INSTALL_ATLAS="true"
 
-    # This environment tests the oldest supported anaconda env
+    # This environment tests Python 2 support
     - DISTRIB="conda" PYTHON_VERSION="2.7" INSTALL_MKL="false"
-      NUMPY_VERSION="1.8.1" SCIPY_VERSION="0.13.3"
-      SKLEARN_VERSION="0.14.1" MATPLOTLIB_VERSION="1.3.1"
+      NUMPY_VERSION="1.9.2" SCIPY_VERSION="0.16.0"
+      SKLEARN_VERSION="0.16cond.1" MATPLOTLIB_VERSION="1.4.3"
```

Comment:
The 2.7 build fails because it doesn't find this sklearn version - are you sure this is the correct version number?